woodwork/tests/type_system/test_ltype_inference.py

```diff
@@ -115,27 +115,25 @@ def test_categorical_inference(categories):
 
 
 def test_categorical_integers_inference(integers):
-    ww.config.set_option('numeric_categorical_threshold', 10)
-    dtypes = ['int8', 'int16', 'int32', 'int64', 'intp', 'int', 'Int64']
-    if _is_koalas_series(integers[0]):
-        dtypes = get_koalas_dtypes(dtypes)
-    for series in integers:
-        for dtype in dtypes:
-            inferred_type = ww.type_system.infer_logical_type(series.astype(dtype))
-            assert isinstance(inferred_type, Categorical)
-    ww.config.reset_option('numeric_categorical_threshold')
+    with ww.config.with_options(numeric_categorical_threshold=10):
+        dtypes = ['int8', 'int16', 'int32', 'int64', 'intp', 'int', 'Int64']
+        if ks and isinstance(integers[0], ks.Series):
```

Comment:
The merge caused your change to be reverted - need to update to use `_is_koalas_series` again.